tests/integration/conftest.py

```diff
@@ -23,13 +23,18 @@
 from wazuh_testing.tools.simulators.authd_simulator import AuthdSimulator
 from wazuh_testing.tools.simulators.remoted_simulator import RemotedSimulator
 from wazuh_testing.utils import configuration, database, file, mocking, services
-from wazuh_testing.utils.file import remove_file
+from wazuh_testing.utils.file import remove_file, truncate_file, recursive_directory_creation
 from wazuh_testing.utils.manage_agents import remove_agents
+from wazuh_testing.utils.services import control_service
+
+from wazuh_testing.constants.paths.logs import WAZUH_LOG_PATH
+from wazuh_testing.tools.monitors import file_monitor
 
 
 #- - - - - - - - - - - - - - - - - - - - - - - - -Pytest configuration - - - - - - - - - - - - - - - - - - - - - - - - -
 
 
+
```

Comment:
Additional line break, it could be removed to preserve the spacing of the rest of the file.